app/graphql/posts/search.graphql

```diff
@@ -23,7 +23,7 @@ query search(
         { name: "creator_id", value: $creator_id }
         { name: "title", value: $title }
         { name: "body", value: $body }
-        { name: "tags", array_contains: $tags }
+        { name: "tags", contains: $tags }
```

Comment:
@hosszukalman since `tags` is an array, we should use one of the array filters, I believe `array_overlaps` is reasonable, for the record pasting documentation for all relevant:
```
        argument :value_in, [String], 'selects records for which the property is included as a sub-array or is an element of the provided Array argument', required: false
        argument :array_contains, [String], 'selects records for which the property of type Array contains all of the provided elements', required: false
        argument :array_overlaps, [String], 'selects records for which the property of type Array contains at least one of the provided elements', required: false
```